submissions/okkkko/js_oop/app.js

```diff
@@ -3,13 +3,13 @@ const STEP_Y = 80;
 const STEP_X = 102;
 const BUGS_PADDING = 18;
 const BUGS_START_POSITION = {
-  x: -200,
-  y: [1, 3, 4].map((rowNumber) => rowNumber * STEP_Y - BUGS_PADDING),
+  x: -200
 };
 const MIN_SPEED = 100;
 const MAX_SPEED = 700;
 const CANVAS_WIDTH = 505;
```

Comment:
But what you will do if width change ? you will need to change this value. If you have access to dynamically calculated value - use it instead of this one.
 
[Old comment](https://github.com/kottans/frontend-2021-homeworks/pull/353#discussion_r549315352)